baybe/utils/dataframe.py

```diff
@@ -413,3 +414,42 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_printing_dataFrame(df: pd.DataFrame) -> str:
```

Comment:
Two issues here:
* this is not snake case
* should be imperative style

```suggestion
def pretty_print_dataframe(df: pd.DataFrame) -> str:
```

Reply:
Fixed. Thank you 😃

Reply:
```suggestion
def pretty_print_df(df: pd.DataFrame) -> str:
```

which is in line with names of other utilities (and shorter and equally understandable)

Reply:
@AdrianSosic Is `pretty_print_df()` okay for you too?